actions/npm_publish_version.meta.yaml

```diff
@@ -0,0 +1,25 @@
+---
+  name: "npm_publish_version"
+  runner_type: "action-chain"
+  description: "Published new versions to npm."
+  enabled: true
+  entry_point: "workflows/npm_publish_version.yaml"
+  parameters:
+    repo:
+      type: "string"
+      description: "Url of the repo to clone"
+      default: "git@github.com:StackStorm/hubot-stackstorm.git"
+      required: true
+    branch:
```

Comment:
We ain't got no release branch here and it does not really matter from the npm perspective. 

Basically, I am trying to only capture the process I follow manually and that works on master. If we change the process then this workflow can be modified to match the process. For now though we ship from master!